go/exorelay/glide.lock

```diff
@@ -1,16 +1,13 @@
-hash: 9eae94d4bbd95fdb3b6aae2aa6e6c5797ba1a0b44f2c4ff383331ff9e930a1c3
-updated: 2017-05-10T08:47:00.908706416-07:00
+hash: 5cfdc0100bae114d0f09b64a1c2594e2273c3dbcd4d3a924c022fd2822a196b6
+updated: 2017-05-16T14:31:11.937662707-07:00
 imports:
 - name: github.com/DATA-DOG/godog
-  version: 38e3d1fb6df6a7b6c59c120ecd02141764d2c343
+  version: 92fbee719c9593813b24a6ef042bcad40324ede4
   subpackages:
   - colors
   - gherkin
-- name: github.com/Originate/exocom
-  version: 30c7f513ced53c2817f0f41aac60322e7eeef069
-  subpackages:
-  - go/exorelay
-  - go/structs
```

Comment:
We don't want these to actually get installed. Glide will add them automatically when you get a new package. I've just manually deleted them.